Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -29,8 +29,27 @@ class LaunchCoordinator: BaseCoordinator,
     func start(with launchType: LaunchType) {
         let isFullScreen = launchType.isFullScreenAvailable(isIphone: isIphone)
         switch launchType {
-        case .intro(let manager):
+        /* Ecosia: Change to support `OnboardingCardNTPExperiment` conditions
+         case .intro(let manager):
             presentIntroOnboarding(with: manager, isFullScreen: isFullScreen)
+         */
+        case .intro(let manager, let checkExperiment):
+            guard checkExperiment else {
+                presentIntroOnboarding(with: manager, isFullScreen: isFullScreen)
+                return
+            }
+            // TODO: Refactor `FeatureManagement.fetchConfiguration()` pre-condition - maybe a notification from FeatureManagement?
+            Task {
+                await FeatureManagement.fetchConfiguration()
+                DispatchQueue.main.async { [weak self] in
+                    guard let self = self else { return }
+                    guard !OnboardingCardNTPExperiment.isEnabled else {
+                        self.parentCoordinator?.didFinishLaunch(from: self)
+                        return
+                    }
+                    self.presentIntroOnboarding(with: manager, isFullScreen: isFullScreen)
+                }
+            }
```

Comment:
For now this is working, but I don't think it looks great as it refreshes Unleash one extra time and mixes two concurrency patterns.

Maybe this could be some sort of Notification sent by `FeatureManagement` whenever it is finished so that places in the code that depend on it can use to make sure it is initialised.

Reply:
We'll need to think about it thoroughly. For the time being, given the experiment context, I thought about leaving it as is. As a next step, I made a DRAFT RFC in our Confluence page that will be document with a proposed solution for us to comment on and eventually apply.